Sources/CovidCertificateSDK/TrustList/TrustListManager.swift

```diff
@@ -183,14 +183,24 @@ class TrustListUpdate {
 
     private func startForceUpdate() {
         internalQueue.sync {
-            let error = synchronousUpdate(ignoreLocalCache: true)
-            // Only reset lastError if synchronousUpdate was successful
-            if error == nil {
```

Comment:
Setting `self.lastError` should still be done on the `operationQueue`, to make sure this variable is synchronized and only read/written from this queue.